packages/buildcop/buildcop.go

```diff
@@ -191,9 +228,9 @@ func processLog(ctx context.Context, repo, installationID, commit string, topic
 			Name:         "buildcop",
 			Type:         "function",
 			Location:     "us-central1",
-			Installation: githubInstallation{ID: installationID},
-			Repo:         repo,
-			Commit:       commit,
+			Installation: githubInstallation{ID: cfg.installationID},
+			Repo:         cfg.repo,
+			Commit:       cfg.commit,
 			BuildURL:     buildURL,
```

Comment:
cfg.buildURL?

Reply:
It's not plumbed through the `config` type yet. I removed it from the struct.